src/syncing.md

```diff
@@ -52,9 +52,9 @@ and red when a full sync is required.
 Related video: <https://www.youtube.com/watch?v=phP9GGG-PxY>
 
 Anki will synchronize any sounds and images used by your notes. It will
-notice when media has been added or removed from your [media folder](files.md#file-locations), but
+notice when media has been added, removed or replaced in your [media folder](files.md#file-locations), but
 will not notice if you have edited some existing files without adding or
```

Comment:
From here make this:

`will not notice if you have made edits to existing files. To get your edits synced, you need to add, remove or replace a
file as well.`